Tasks/AppCenterDistributeV1/Tests/L0PublishCommitInfo_1.ts

```diff
@@ -4,14 +4,15 @@ import tmrm = require('vsts-task-lib/mock-run');
 import path = require('path');
 import fs = require('fs');
 import azureBlobUploadHelper = require('../azure-blob-upload-helper');
+import { basicSetup, mockFs } from './TestHelpers';
```

Comment:
For mocking you can use packages like sinon - it could be much easier, no need to write them manually

Reply:
Answered [here](https://github.com/microsoft/azure-pipelines-tasks/pull/13204#discussion_r448810257)